modules/apps/map/map-common/test/liferay/MapBase.es.js

```diff
@@ -547,11 +577,16 @@ describe('MapBase', () => {
 
 	describe('_handleSearchButtonClicked()', () => {
 		it('updates the instance position', () => {
-			const position = {location: getLocation()};
+			const location = getLocation();
 
-			mapImpl._handleSearchButtonClicked({position});
+			mapImpl._handleSearchButtonClicked(location);
```

Comment:
This looks like a different implementation of this function...

`{position: {location: {lat: number, lng: number}}}` !== `{lat: number, lng: number}`